src/stories/pages/404.stories.tsx

```diff
@@ -5,14 +5,14 @@ import FourOhFourComponent from '~js/components/404';
 
 export default {
   title: 'Pages/404/Component',
+  component: FourOhFourComponent,
 };
 
 const Template: Story<ComponentProps<typeof FourOhFourComponent>> = (args) => (
   <FourOhFourComponent {...args} />
 );
 
 export const FourOhFour = Template.bind({});
-
 FourOhFour.args = {
   message: 'Lorem Ipsum',
```

Comment:
@jgerigmeyer not a big deal for this pr, but I'm wondering if there's another decorator missing to account for components that rely on having the `routes` available like this 404 if we called this without a `message` arg.

Reply:
In `preview.js`, we're using the `<MemoryRouter>` in the global decorator, which allows using react-router things (e.g. `<Link>`) in our stories -- so this works as-is.